### PR TITLE
PD-84 Update Replication Tutorials

### PR DESCRIPTION
**Pull request by micjohnson777**
This initial commit moves content form the AddReplicationSCALE.md article to the index and reworks it. It deletes the AddReplicationSCALE.md file. It also creates an overview procedure snippet to use in the other tutorials.

It also creates a snippet of the zvol storage warning and moves that warning from the AddManageZvols.md article, replaces the text with the snippet, and adds this snippet to the Local Replication tutorial.

Thanks for contributing to TrueNAS documentation! By opening a Pull Request, you're acknowledging that your changes will be distributed under the [Creative Commons 4.0](https://creativecommons.org/licenses/by-nc-sa/4.0/) license.

### CODE REVIEW

File: content/SCALE/SCALETutorials/DataProtection/Replication/ReplicationWithEncryptionSCALE.md [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+
+3. Create an SSH connection between the local SCALE system and the remote system. 
+   You can do this from either **Credentials > Backup Credentials > SSH Connection** and clicking **Add** or from the **Replication Task Wizard** using the **Generate New** option in the settings for the remote system.
+
+4. Unlock the encrypted dataset(s) and export the encryption key to a text editor like Notepad.
+
+5. Go to **Data Protection > Replication Tasks** and click **Add** to open the **Replication Task Wizard**. 
+   You then specify the from and to sources, task name, and set the schedule.
+  
+   Setting options change based on the source selections. Replicating to or from a local source does not requires an SSH connection.
```
Comment (markbainix) [inline content/SCALE/SCALETutorials/DataProtection/Replication/ReplicationWithEncryptionSCALE.md:50]: Suggest reword: 'Replicating to or from a local source..." to "A to or from source that is local, does not..." the first one almost sounds like any time you are replicating to or from a local source (which is always) you never need an SSH connection. Or perhaps "When a source (to or from) is remote, an SSH Connection is required"

File: content/SCALE/SCALETutorials/DataProtection/Replication/ReplicationWithEncryptionSCALE.md [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+
+4. Go to **Data Protection** and click **Add** on the **Replication Tasks** widget to open the **Replication Task Wizard**. Configure the following settings:
+   
+   ![CreateRemoteReplicationTask](/images/SCALE/22.12/CreateRemoteReplicationTask.png "New Remote Replication Task")
+   
+   a. Select either **On this System** or **On a Different System** on the **Source Location** dropdown list. 
+      If your source is a remote system, select **On a Different System**. The **Destination Location** automatically changes to **On this System**.       
+      If your source is the local TrueNAS SCALE system, you must select **On a Different System** from the **Destination Location** dropdown list to do remote replication. 
+      
+      TrueNAS shows the number snapshots available for replication.
```
Comment (markbainix) [inline content/SCALE/SCALETutorials/DataProtection/Replication/ReplicationWithEncryptionSCALE.md:78]: missing word: 'number **of** snapshots'

File: content/SCALE/SCALETutorials/DataProtection/Replication/ReplicationWithEncryptionSCALE.md [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+    
+    b. Select an existing SSH connection to the remote system.
```
Comment (markbainix) [inline content/SCALE/SCALETutorials/DataProtection/Replication/ReplicationWithEncryptionSCALE.md:80]: should this be an 'or'? or reword to suggest you do one or the other. currently it looks like you do both.

File: content/SCALE/SCALETutorials/DataProtection/Replication/ReplicationWithEncryptionSCALE.md [hunk 4]
```diff
@@ -0,0 +1,142 @@
...
+       Select **Create New** to open the **[New SSH Connection](#configure-a-new-ssh-connection)** configuration screen.
+    
+    c. Browse to the source pool/dataset(s), then click on the dataset(s) to populate the **Source** with the path. 
+       You can select multiple sources or manually type the names into the **Source** field.
```
Comment (markbainix) [inline content/SCALE/SCALETutorials/DataProtection/Replication/ReplicationWithEncryptionSCALE.md:84]: might want to add after 'manually type.. field.' the following '(separated by commas)'

File: content/SCALE/SCALETutorials/DataProtection/Replication/ReplicationWithEncryptionSCALE.md [hunk 5]
```diff
@@ -0,0 +1,142 @@
...
+       Remote sources require entering a snapshot naming schema to identify the snapshots to replicate. 
+       A naming schema is a pattern of naming custom snapshots you want to replicate. 
+       If you want to change the default schema, enter the name and [strftime(3)](https://man7.org/linux/man-pages/man3/strftime.3.html) %Y, %m, %d, %H, and %M strings that match the snapshots to include in the replication. 
+       Separate entries by pressing <kbd>Enter</kbd>. The number of snapshots matching the patterns display.
+    
+    ![RemoteReplicateSnapshotAndNameSchema](/images/SCALE/22.12/RemoteReplicateSnapshotAndNameSchema.png "Replicate Custom Snapshot and Naming Schema")
+
+   h. (Optional) Enter a name for the snapshot in **Task Name**. 
+      SCALE populates this field with the default name using the source and destination paths separated by a hyphen, but this default can make locating the snapshot in destination dataset a challenge. 
+      To make it easier to find the snapshot, give it name easy for you to identify. For example, a replicated task named *dailyfull* for a full file system snapshot taken daily.
```
Comment (markbainix) [inline content/SCALE/SCALETutorials/DataProtection/Replication/ReplicationWithEncryptionSCALE.md:110]: Missing a word: 'give it name' to 'give it a name'
Also, 'name easy for you to identify' to 'name for you to easily identify'

File: content/SCALE/SCALETutorials/DataProtection/Replication/_index.md [hunk 6]
```diff
@@ -1,10 +1,34 @@
...
 geekdocCollapseSection: true
+aliases:
+ - /scale/scaletutorials/dataprotection/addreplicationscale/
+ - /scale/scaletutorials/dataprotection/replication/addreplicationscale/
 weight: 100
 ---
 
 
+TrueNAS SCALE replication allows users to create one-time or regularly scheduled snapshots of data stored in pools, datasets or zvols on their SCALE system as a way to back up stored data. 
+When properly configured and scheduled, replication takes take regular snapshots of storage pools or datasets and saves them in the destination location either on the same system or a different system.
```
Comment (markbainix) [inline content/SCALE/SCALETutorials/DataProtection/Replication/_index.md:12]: extra word: 'remote replication takes **take** regular...'

File: content/_includes/ReplicationIntroSCALE.md [hunk 7]
```diff
@@ -0,0 +1,11 @@
+---
+---
+
+The first snapshot taken for a task creates a full file system snapshot, and all subsequent snapshots taken for that task are incremental to capture differences occurring between the full and subsequent incremental snapshots.
+
+Scheduling options allow users to run replication tasks daily, weekly, monthly, or on a custom schedule. 
+Users also have the option to run a scheduled job on demand. 
+{{< hint info >}}
+Replication tasks require a periodic snapshot task. Early release of SCALE required you to create a periodic snapshot task before you create the replication task, but SCALE now creates this task before it runs a replication task according to a schedule.
```
[the remark this answers is not in the document]
Comment (micjohnson777) [reply content/_includes/ReplicationIntroSCALE.md:9]: nope, it is only the early or .1 release so not earlier bluefin releases.

File: content/SCALE/SCALETutorials/DataProtection/Replication/LocalReplicationSCALE.md [hunk 8]
```diff
@@ -1,87 +1,65 @@
...
-  * Define a **Destination** path
-    * Set the destination location to the local system
-  	 * Select or manually define a path to the single destination location for the snapshot copies.
-  * Set the **Replication schedule** to run once
-  * Define how long the snapshots are stored in the **Destination**
-  * Clicking **START REPLICATION** immediately snapshots the chosen sources and copies those snapshots to the destination
-    * Dialog might ask to delete existing snapshots from the destination. Be sure that all important data is protected before deleting anything.
-* Clicking the task **State** shows the logs for that replication task.
-{{< /expand >}}
+Local replication on systems with more than one pool can also create a dataset to use for the replicated snapshots.
```
[the remark this answers is not in the document]
Comment (micjohnson777) [reply content/SCALE/SCALETutorials/DataProtection/Replication/LocalReplicationSCALE.md:25]: good point. I'll smooth this out

File: content/_includes/BasicReplicationProcess.md [hunk 9]
```diff
@@ -0,0 +1,22 @@
...
+---
+
+{{< expand "Replication Task General Overview" "v" >}}
+
+1. Set up the data storage for where you want to save replicated snapshots. 
+   
+2. Make sure the admin user has a home directory assigned. 
+   In the SCALE Bluefin early release, when creating the admin user at installation the home directory default is set to **/nonexistent**. To create an SSH connection to use in a remote replication you must assign a home directory path.
+
+   Later releases of SCALE Bluefin set the admin user home directory to one created by SCALE during the installation process, but you need to select the option to create the admin user home directory.
```
[the remark this answers is not in the document]
Comment (micjohnson777) [reply content/_includes/BasicReplicationProcess.md:11]: nope. we are not using checkbox if at all possible so we say option or setting.

File: content/_includes/BasicReplicationProcess.md [hunk 10]
```diff
@@ -0,0 +1,22 @@
...
+
+3. Create an SSH connection between the local SCALE system and the remote system.
```
[the remark this answers is not in the document]
Comment (micjohnson777) [reply content/_includes/BasicReplicationProcess.md:13]: ssh is only required for remote, ,but there are always a source and destination. if you select On this System as source you can also make destination the same but if you select On a Different System, the other whether source or destination, that automatically gets set to On this System. I'll smooth this out

File: content/_includes/BasicReplicationProcess.md [hunk 11]
```diff
@@ -0,0 +1,22 @@
...
+
+   Later releases of SCALE Bluefin set the admin user home directory to one created by SCALE during the installation process, but you need to select the option to create the admin user home directory.
+
+3. Create an SSH connection between the local SCALE system and the remote system. 
+   You can do this from either **Credentials > Backup Credentials > SSH Connection** and clicking **Add** or from the **Repliation Task Wizard** using the **Generate New** option in the settings for the remote system.
+
+4. Go to **Data Protection > Replication Tasks** and click **Add** to open the **Replication Task Wizard**. 
+   You then specify the from and to sources, task name, and set the schedule.
+  
+   Setting options change based on the source selections. Replicating to or from a local source does not requires an SSH connection.
```
[the remark this answers is not in the document]
Comment (micjohnson777) [reply content/_includes/BasicReplicationProcess.md:19]: local replication does not use or require an ssh connection but remote replication does require this. I'll reword to smooth this out

File: content/_includes/ReplicationIntroSCALE.md (hunk 7 above)
[the remark this answers is not in the document]
Comment (micjohnson777) [reply content/_includes/ReplicationIntroSCALE.md:9]: Resolved this to be more general.

**Review by markbainix — APPROVED** [review]

File: content/SCALE/SCALETutorials/DataProtection/Replication/LocalReplicationSCALE.md [hunk 12]
```diff
@@ -1,87 +1,64 @@
...
 
 
-## Local Replication
+## Using Local Replication
 
-### Process Summary
+A local replication creates a zfs snapshot and saves it to another location on the same TrueNAS SCALE system either using a different pool, or dataset or zvol. 
+This allows users with only one system to take quick data backups or snapshots of their data when they have only one system.
+In this scenario, create a dataset on the same pool to store the replication snapshots. You can create and use a zvol for this purpose.
+If configuring local replication on a system with more than one pool, create a dataset to use for the replicated snapshots on one of those pools.
```
Comment (tonyriv3) [inline content/SCALE/SCALETutorials/DataProtection/Replication/LocalReplicationSCALE.md:20]:
``` suggestion
A local replication creates a zfs snapshot and saves it to another location on the same TrueNAS SCALE system using either a different pool, dataset, or zvol. 
This allows users with only one system to take quick data backups or snapshots of their data.
In this scenario, create a dataset on the same pool to store the replication snapshots. You can create and use a zvol for this purpose.
If configuring local replication on a system with more than one pool, create a dataset for the replicated snapshots on one of those pools.
```

File: content/SCALE/SCALETutorials/DataProtection/Replication/LocalReplicationSCALE.md [hunk 13]
```diff
@@ -1,87 +1,64 @@
...
-{{< /expand >}}
+With the implementation of rootless login and the admin user, setting up replication tasks as an admin user has a few differences over setting up replication tasks when logged in as root. 
 
-### Quick Local Backups with the Replication Wizard
+{{< include file="/content/_includes/ReplicationIntroSCALE.md" type="page" >}}
 
-TrueNAS provides a wizard for quickly configuring different simple replication scenarios.
+## Setting Up a Simple Replication Task Overview 
+This section provides a simple overview of setting up a replication task regardless of the type of replication, local or remote. 
+It also covers the related steps you should take prior to configuring a replication task.
```
Comment (tonyriv3) [inline content/SCALE/SCALETutorials/DataProtection/Replication/LocalReplicationSCALE.md:32]:
```suggestion
This section provides a simple overview of setting up a replication task regardless of the type of replication, local or remote. 
It also covers the steps you should take before configuring a replication task. 
```

File: content/SCALE/SCALETutorials/DataProtection/Replication/LocalReplicationSCALE.md [hunk 14]
```diff
@@ -1,87 +1,64 @@
...
 
-To open the **Replication Wizard**, go to **Data Protection > Replication Tasks** and click **ADD**.
-Set the source location to the local system and pick which datasets to snapshot.
-The wizard takes new snapshots of the sources when no existing source snapshots are found.  
-Enabling **Recursive** replicates all snapshots contained within the selected source dataset snapshots.
-Local sources can also use a naming schema to identify any custom snapshots to include in the replication.
-A naming schema is a collection of [strftime](https://www.freebsd.org/cgi/man.cgi?query=strftime) time and date strings and any identifiers that a user might have added to the snapshot name.
+If you have an existing replication task, you can select it on the **Load Previous Replication Task** dropdown list to load the configuration settings for that task into the wizard, and then make change such as assigning it a different destination, schedule, or retention lifetime, etc. 
+Saving changes to the configuration creates a new replication task without altering the task you loaded into the wizard.
```
Comment (tonyriv3) [inline content/SCALE/SCALETutorials/DataProtection/Replication/LocalReplicationSCALE.md:41]:
```suggestion
If you have an existing replication task, you can select it on the **Load Previous Replication Task** dropdown list to load the configuration settings for that task into the wizard, then make changes such as assigning it a different destination, schedule, retention lifetime, etc. 
Saving changes to the configuration creates a new replication task without altering the task you loaded into the wizard.
```

File: content/SCALE/SCALETutorials/DataProtection/Replication/LocalReplicationSCALE.md [hunk 15]
```diff
@@ -1,87 +1,64 @@
...
 
-![TasksReplicationTasksAddLocalDestSCALE](/images/SCALE/RepWizardLocalDestSCALE.png "Local Destination")
+   b. Select **On this System** on the **Destination Location** dropdown list. 
+      Browse to the location of the pool or dataset you want to use to store replicated snapshots and select it so it populates the **Destination** with the path.
 
-TrueNAS suggests a default name for the task based on the selected source and destination locations, but you can type your own name for the replication.
-You can load any saved replication task into the wizard to make creating new replication schedules even easier.
+   c. (Optional) Enter a name for the snapshot in **Task Name**. 
+      SCALE populates this field with the default name using the source and destination paths separated by a hyphen, but this default can make locating the snapshot in destination dataset a challenge. 
+      To make it easier to find the snapshot, give it name easy for you to identify. For example, a replicated task named *dailyfull* for a full file system snapshot taken daily.
```
Comment (tonyriv3) [inline content/SCALE/SCALETutorials/DataProtection/Replication/LocalReplicationSCALE.md:58]:
```suggestion
   c. (Optional) Enter a name for the snapshot in **Task Name**. 
      SCALE populates this field with the default name using the source and destination paths separated by a hyphen, which can make locating the snapshot in the destination dataset challenging. 
      To make it easier to find the snapshot, give it an easily identifiable name. For example, a replicated task named *dailyfull* for a full file system snapshot taken daily. 
```

File: content/SCALE/SCALETutorials/DataProtection/Replication/RemoteReplicationSCALE.md [hunk 16]
```diff
@@ -1,112 +1,89 @@
...
-Choosing a remote destination requires an SSH connection to that system.
-Expanding the directory browser shows the current datasets that are available for replication.
-You can select a destination dataset or manually type a path in the field.
-You cannot use zvols as a remote replication destination.
-Adding a name to the end of the path creates a new dataset in that location.
+## Using Remote Replication
 
-![TasksReplicationTasksAddRemoteDestSCALE](/images/SCALE/RepAddDestinationSCALE.png "Replication with Remote Destination")
+TrueNAS SCALE replication allows users to create one-time or regularly scheduled snapshots of data stored in pools, datasets or zvols on their SCALE system as a way to back up stored data. 
+When properly configured and scheduled, remote replication takes take regular snapshots of storage pools or datasets and saves them in the destination location on another system.
```
Comment (tonyriv3) [inline content/SCALE/SCALETutorials/DataProtection/Replication/RemoteReplicationSCALE.md:17]:
```suggestion
TrueNAS SCALE replication allows users to create one-time or regularly-scheduled snapshots of data stored in pools, datasets, or zvols on their SCALE system to back up stored data. 
When properly configured and scheduled, remote replication takes regular snapshots of storage pools or datasets and saves them in the destination location on another system. 
```

File: content/SCALE/SCALETutorials/DataProtection/Replication/RemoteReplicationSCALE.md [hunk 17]
```diff
@@ -1,112 +1,89 @@
...
-Adding a name to the end of the path creates a new dataset in that location.
+## Using Remote Replication
 
-![TasksReplicationTasksAddRemoteDestSCALE](/images/SCALE/RepAddDestinationSCALE.png "Replication with Remote Destination")
+TrueNAS SCALE replication allows users to create one-time or regularly scheduled snapshots of data stored in pools, datasets or zvols on their SCALE system as a way to back up stored data. 
+When properly configured and scheduled, remote replication takes take regular snapshots of storage pools or datasets and saves them in the destination location on another system. 
 
-{{< hint info >}}
-To use encryption when replicating data click the **Encryption** box. After selecting the box these additional encryption options  become available:
+Remote replication can occur between your TrueNAS SCALE system and another TrueNAS system (SCALE or CORE) where you want to use to store your replicated snapshots.
```
Comment (tonyriv3) [inline content/SCALE/SCALETutorials/DataProtection/Replication/RemoteReplicationSCALE.md:19]:
```suggestion
Remote replication can occur between your TrueNAS SCALE system and another TrueNAS system (SCALE or CORE) where you want to store your replicated snapshots. 
```

File: content/SCALE/SCALETutorials/DataProtection/Replication/RemoteReplicationSCALE.md [hunk 18]
```diff
@@ -1,112 +1,89 @@
...
 
-{{< hint info >}}
-To use encryption when replicating data click the **Encryption** box. After selecting the box these additional encryption options  become available:
+Remote replication can occur between your TrueNAS SCALE system and another TrueNAS system (SCALE or CORE) where you want to use to store your replicated snapshots. 
 
-* **Encryption Key Format** allows the user to choose between a hex (base 16 numeral) or passphrase (alphanumeric) style encryption key. <br>
-* **Store Encryption key in Sending TrueNAS database** allows the user to either store the encryption key in the sending TrueNAS database (box checked) or choose a temporary location for the encryption key that decrypts replicated data (box unchecked)
-{{< /hint >}}
+With the implementation of rootless login and the admin user, setting up replication tasks as an admin user has a few differences than with setting up replication tasks when logged in as root. Setting up remote replication while logged in as the admin user requires selecting **Use Sudo For ZFS Commands**.
```
Comment (tonyriv3) [inline content/SCALE/SCALETutorials/DataProtection/Replication/RemoteReplicationSCALE.md:21]:
```suggestion
With the implementation of rootless login and the admin user, setting up replication tasks as an admin user differs from setting up replication tasks when logged in as root. Setting up remote replication while logged in as the admin user requires selecting **Use Sudo For ZFS Commands**. 
```

File: content/SCALE/SCALETutorials/DataProtection/Replication/RemoteReplicationSCALE.md [hunk 19]
```diff
@@ -1,112 +1,89 @@
...
-### Security and Task Name
+Remote replication requires setting up an SSH connection in TrueNAS before creating a remote replication task. 
 
-{{< expand "Security and Task Name" "v" >}}
-{{< hint info >}}
-Using encryption for SSH transfer security is always recommended.
-{{< /hint >}}
+## Setting Up a Simple Replication Task Overview 
+This section provides a simple overview of setting up a replication task regardless of the type of replication, local or remote. 
+It also covers the related steps you should take prior to configuring a replication task.
```
Comment (tonyriv3) [inline content/SCALE/SCALETutorials/DataProtection/Replication/RemoteReplicationSCALE.md:29]:
```suggestion
This section provides a simple overview of setting up a replication task regardless of the type of replication, local or remote. 
It also covers the steps you should take before configuring a replication task. 
```

File: content/SCALE/SCALETutorials/DataProtection/Replication/RemoteReplicationSCALE.md [hunk 20]
```diff
@@ -1,112 +1,89 @@
...
-New tasks are enabled by default and activate according to their schedule or immediately when no schedule is chosen.
-The first time a replication task runs, it takes longer because the snapshots must be copied entirely fresh to the destination.
-
-![TasksReplicationTasksSuccessSCALE](/images/SCALE/RepSuccessSCALE.png "Remote Replication Success")
+## Creating a Remote Replication Task
 
-Later replications run faster since the task only replicates subsequent changes to snapshots.
-Clicking the task state opens the log for that task.
+To streamline creating simple replication tasks use the **Replication Task Wizard** to create and copy ZFS snapshots to another system. 
+The wizard assists with creating a new SSH connection and automatically creates a periodic snapshot task for sources that have no existing snapshots.
```
Comment (tonyriv3) [inline content/SCALE/SCALETutorials/DataProtection/Replication/RemoteReplicationSCALE.md:36]:
```suggestion
To streamline creating simple replication tasks, use the **Replication Task Wizard** to create and copy ZFS snapshots to another system. 
The wizard helps create a new SSH connection and automatically creates a periodic snapshot task for sources with no existing snapshots.
```

File: content/SCALE/SCALETutorials/DataProtection/Replication/RemoteReplicationSCALE.md [hunk 21]
```diff
@@ -1,112 +1,89 @@
...
 
-![TasksReplicationTasksLogSCALE](/images/SCALE/RepLogSCALE.png "Replication Log")
+If you have an existing replication task, you can select it on the **Load Previous Replication Task** dropdown list to load the configuration settings for that task into the wizard, and then make change such as assigning it a different destination, schedule, or retention lifetime, etc.
```
Comment (tonyriv3) [inline content/SCALE/SCALETutorials/DataProtection/Replication/RemoteReplicationSCALE.md:38]:
```suggestion
If you have an existing replication task, you can select it on the **Load Previous Replication Task** dropdown list to load the task configuration settings into the wizard and then make changes, such as assigning it a different destination, schedule, retention lifetime, etc. 
```

File: content/SCALE/SCALETutorials/DataProtection/Replication/RemoteReplicationSCALE.md [hunk 22]
```diff
@@ -1,112 +1,89 @@
...
+
+3. Go to **Data Protection** and click **Add** on the **Replication Tasks** widget to open the **Replication Task Wizard**. Configure the following settings:
+   
+   ![CreateRemoteReplicationTask](/images/SCALE/22.12/CreateRemoteReplicationTask.png "New Remote Replication Task")
+   
+   a. Select either **On this System** or **On a Different System** on the **Source Location** dropdown list. 
+      If your source is a remote system, select **On a Different System**. The **Destination Location** automatically changes to **On this System**.       
+      If your source is the local TrueNAS SCALE system, you must select **On a Different System** from the **Destination Location** dropdown list to do remote replication. 
+      
+      TrueNAS shows the number snapshots available for replication.
```
Comment (tonyriv3) [inline content/SCALE/SCALETutorials/DataProtection/Replication/RemoteReplicationSCALE.md:52]:
```suggestion
      TrueNAS shows the number of snapshots available for replication.
```

File: content/SCALE/SCALETutorials/DataProtection/Replication/RemoteReplicationSCALE.md [hunk 23]
```diff
@@ -1,112 +1,89 @@
...
+  
+   e. Select **Use Sudo for ZFS Commands**. Only displays when logged in as the admin user (or the name of the admin user). 
+      This removes the need to issue the cli `zfs allow` command in Shell on the remote system. 
+
+   ![CreateRemoteReplicationTaskSetSudo](/images/SCALE/22.12/CreateRemoteReplicationTaskSetSudo.png "Select Use Sudo for ZFS Commands")
+    
+   f. Select the **Include snapshots with the name** radio button you want to use. If **Naming Schema**, you must enter a schema in **Naming Schema**.
+      Remote sources require entering a snapshot naming schema to identify the snapshots to replicate. 
+      A naming schema is a pattern of naming custom snapshots you want to replicate. 
+      Enter the name and [strftime(3)](https://man7.org/linux/man-pages/man3/strftime.3.html) %Y, %m, %d, %H, and %M strings that match the snapshots to include in the replication. Separate entries by pressing <kbd>Enter</kbd>. The number of snapshots matching the patterns display.
```
Comment (tonyriv3) [inline content/SCALE/SCALETutorials/DataProtection/Replication/RemoteReplicationSCALE.md:71]:
```suggestion
      Enter the name and [strftime(3)](https://man7.org/linux/man-pages/man3/strftime.3.html) %Y, %m, %d, %H, and %M strings that match the snapshots to include in the replication. Separate entries by pressing <kbd>Enter</kbd>. The number of snapshots matching the patterns displays.
```

File: content/SCALE/SCALETutorials/DataProtection/Replication/RemoteReplicationSCALE.md [hunk 24]
```diff
@@ -1,112 +1,89 @@
...
+
+   g. (Optional) Enter a name for the snapshot in **Task Name**. 
+      SCALE populates this field with the default name using the source and destination paths separated by a hyphen, but this default can make locating the snapshot in destination dataset a challenge. 
+      To make it easier to find the snapshot, give it name easy for you to identify. For example, a replicated task named *dailyfull* for a full file system snapshot taken daily.
```
Comment (tonyriv3) [inline content/SCALE/SCALETutorials/DataProtection/Replication/RemoteReplicationSCALE.md:75]:
```suggestion
   g. (Optional) Enter a name for the snapshot in **Task Name**. 
      SCALE populates this field with the default name using the source and destination paths separated by a hyphen, which can make locating the snapshot in the destination dataset challenging. 
      To make it easier to find the snapshot, give it an easily identifiable name. For example, a replicated task named *dailyfull* for a full file system snapshot taken daily. 
```

File: content/SCALE/SCALETutorials/DataProtection/Replication/RemoteReplicationSCALE.md [hunk 25]
```diff
@@ -1,112 +1,89 @@
...
+      A naming schema is a pattern of naming custom snapshots you want to replicate. 
+      Enter the name and [strftime(3)](https://man7.org/linux/man-pages/man3/strftime.3.html) %Y, %m, %d, %H, and %M strings that match the snapshots to include in the replication. Separate entries by pressing <kbd>Enter</kbd>. The number of snapshots matching the patterns display.
+
+   g. (Optional) Enter a name for the snapshot in **Task Name**. 
+      SCALE populates this field with the default name using the source and destination paths separated by a hyphen, but this default can make locating the snapshot in destination dataset a challenge. 
+      To make it easier to find the snapshot, give it name easy for you to identify. For example, a replicated task named *dailyfull* for a full file system snapshot taken daily. 
+    
+{{< include file="/content/_includes/ReplicationScheduleAndRetentionSteps.md" type="page" >}}
+
+For information on replicating encrypted pools or datasets, see [Setting Up a Encrypted Replication Task]({{< relref "ReplicationWithEncryptionSCALE.md" >}}).
```
Comment (tonyriv3) [inline content/SCALE/SCALETutorials/DataProtection/Replication/RemoteReplicationSCALE.md:79]:
```suggestion
For information on replicating encrypted pools or datasets, see [Setting Up an Encrypted Replication Task]({{< relref "ReplicationWithEncryptionSCALE.md" >}}).
```

File: content/SCALE/SCALETutorials/DataProtection/Replication/ReplicationWithEncryptionSCALE.md [hunk 26]
```diff
@@ -0,0 +1,142 @@
...
+- scalereplication
+- scalebackup
+---
+
+{{< toc >}}
+
+
+## Using Encryption in Replication Tasks
+
+TrueNAS SCALE replication allows users to create replicated snapshots of data stored in encrypted pools, datasets or zvols that on their SCALE system as a way to back up stored data to a remote system. You can use encrypted datasets in a local replication.
```
Comment (tonyriv3) [inline content/SCALE/SCALETutorials/DataProtection/Replication/ReplicationWithEncryptionSCALE.md:16]:
```suggestion
TrueNAS SCALE replication allows users to create replicated snapshots of data stored in encrypted pools, datasets, or zvols on their SCALE system to back up stored data to a remote system. You can use encrypted datasets in a local replication.
```

File: content/SCALE/SCALETutorials/DataProtection/Replication/ReplicationWithEncryptionSCALE.md [hunk 27]
```diff
@@ -0,0 +1,142 @@
...
+
+{{< hint warning >}}
+You can set up a replication task for a dataset encrypted with a passphrase or a hex encryption key, but you must unlock the dataset before the task runs or the task fails.
```
Comment (tonyriv3) [inline content/SCALE/SCALETutorials/DataProtection/Replication/ReplicationWithEncryptionSCALE.md:19]:
```suggestion
You can set up a replication task for a dataset encrypted with a passphrase or a hex encryption key, but you must unlock the dataset before running the task, or it fails.
```

File: content/SCALE/SCALETutorials/DataProtection/Replication/ReplicationWithEncryptionSCALE.md [hunk 28]
```diff
@@ -0,0 +1,142 @@
...
+{{< /hint>}}
+
+With the implementation of rootless login and the admin user, when setting up remote replication tasks when logged in as an admin user requires selecting **Use Sudo For ZFS Commands**.
```
Comment (tonyriv3) [inline content/SCALE/SCALETutorials/DataProtection/Replication/ReplicationWithEncryptionSCALE.md:22]:
```suggestion
With the implementation of rootless login and the admin user, setting up remote replication tasks when logged in as an admin user requires selecting **Use Sudo For ZFS Commands**. 
```

File: content/SCALE/SCALETutorials/DataProtection/Replication/ReplicationWithEncryptionSCALE.md [hunk 29]
```diff
@@ -0,0 +1,142 @@
...
+
+{{< hint warning >}}
+You can set up a replication task for a dataset encrypted with a passphrase or a hex encryption key, but you must unlock the dataset before the task runs or the task fails.
+{{< /hint>}}
+
+With the implementation of rootless login and the admin user, when setting up remote replication tasks when logged in as an admin user requires selecting **Use Sudo For ZFS Commands**. 
+
+{{< include file="/content/_includes/ReplicationIntroSCALE.md" type="page" >}}
+
+Remote replication with datasets also require an SSH connection in TrueNAS. You can use an existing SSH connection if it has the same user credentials you want to use for the new replication task.
```
Comment (tonyriv3) [inline content/SCALE/SCALETutorials/DataProtection/Replication/ReplicationWithEncryptionSCALE.md:26]:
```suggestion
Remote replication with datasets also requires an SSH connection in TrueNAS. You can use an existing SSH connection if it has the same user credentials you want to use for the new replication task. 
```

File: content/SCALE/SCALETutorials/DataProtection/Replication/ReplicationWithEncryptionSCALE.md [hunk 30]
```diff
@@ -0,0 +1,142 @@
...
+With the implementation of rootless login and the admin user, when setting up remote replication tasks when logged in as an admin user requires selecting **Use Sudo For ZFS Commands**. 
+
+{{< include file="/content/_includes/ReplicationIntroSCALE.md" type="page" >}}
+
+Remote replication with datasets also require an SSH connection in TrueNAS. You can use an existing SSH connection if it has the same user credentials you want to use for the new replication task. 
+
+## Setting Up a Simple Replication Task Overview 
+
+This section provides a simple overview of setting up a remote replication task for an encrypted dataset. 
+It also covers the related steps you should take prior to configuring the replication task.
```
Comment (tonyriv3) [inline content/SCALE/SCALETutorials/DataProtection/Replication/ReplicationWithEncryptionSCALE.md:31]:
```suggestion
This section provides a simple overview of setting up a remote replication task for an encrypted dataset. 
It also covers the steps you should take before configuring the replication task. 
```

File: content/SCALE/SCALETutorials/DataProtection/Replication/ReplicationWithEncryptionSCALE.md [hunk 31]
```diff
@@ -0,0 +1,142 @@
...
+  
+   Setting options change based on the source selections. Replicating to or from a local source does not requires an SSH connection.
+   
+This completes the general process for all replication tasks.
+{{< /expand >}}
+
+## Creating a Remote Replication Task for an Encrypted Dataset
+
+To streamline creating simple replication tasks use the **Replication Task Wizard** to create and copy ZFS snapshots to another system. 
+The wizard assists with creating a new SSH connection and automatically creates a periodic snapshot task for sources that have no existing snapshots.
```
Comment (tonyriv3) [inline content/SCALE/SCALETutorials/DataProtection/Replication/ReplicationWithEncryptionSCALE.md:58]:
```suggestion
To streamline creating simple replication tasks, use the **Replication Task Wizard** to create and copy ZFS snapshots to another system. 
The wizard assists with creating a new SSH connection and automatically creates a periodic snapshot task for sources with no existing snapshots.
```

File: content/SCALE/SCALETutorials/DataProtection/Replication/ReplicationWithEncryptionSCALE.md [hunk 32]
```diff
@@ -0,0 +1,142 @@
...
+
+If you have an existing replication task, you can select it on the **Load Previous Replication Task** dropdown list to load the configuration settings for that task into the wizard, and then make change such as assigning it a different destination, select encryption options, schedule, or retention lifetime, etc. 
+Saving changes to the configuration creates a new replication task without altering the task you loaded into the wizard.
+This saves some time when creating multiple replication tasks between the same two systems.
```
Comment (tonyriv3) [inline content/SCALE/SCALETutorials/DataProtection/Replication/ReplicationWithEncryptionSCALE.md:62]:
```suggestion
If you have an existing replication task, you can select it on the **Load Previous Replication Task** dropdown list to load the configuration settings for that task into the wizard, then make changes such as assigning it a different destination, selecting encryption options, schedule, retention lifetime, etc. 
Saving changes to the configuration creates a new replication task without altering the task you loaded into the wizard.
This saves some time when creating multiple replication tasks between the same two systems.
```

File: content/SCALE/SCALETutorials/DataProtection/Replication/ReplicationWithEncryptionSCALE.md [hunk 33]
```diff
@@ -0,0 +1,142 @@
...
+
+{{< include file="/content/_includes/ReplicationCreateDatasetAndAdminHomeDirSteps.md" type="page" >}}
+
+3. Unlock the source dataset and export the encryption key to a text editor such as Notepad.
+   Go to **Datasets** select the source dataset, locate the **ZFS Encryption** widget and unlock the dataset if locked. 
+   Export the key and paste it in any text editor such as Notepad. If you set up encryption to use a passphrase, you do not need to export a key.
```
Comment (tonyriv3) [inline content/SCALE/SCALETutorials/DataProtection/Replication/ReplicationWithEncryptionSCALE.md:68]:
```suggestion
3. Unlock the source dataset and export the encryption key to a text editor like Notepad.
   Go to **Datasets**, select the source dataset, locate the **ZFS Encryption** widget, and unlock the dataset if locked. 
   Export the key, then paste it into any text editor, like Notepad. You do not need to export a key if you set up encryption to use a passphrase.
```

**Pull request merged**